src/index.ts

```diff
@@ -130,7 +130,7 @@ if (localStorage.backendDomain && localStorage.backendDomain !== ${JSON.stringif
 localStorage.backendDomain = ${JSON.stringify(info.backend)};
 if (
 	(localStorage.auth === 'null' && localStorage.prevAuth === 'null') ||
-	!(Date.now() - localStorage.lastToken < 2 * 60000) ||
+	60 * 60 * 1000 < Date.now() - localStorage.lastToken ||
```

Comment:
I suspect this change only affects the timeout when going through the user registration to create a new account (for example if you take longer than 2 minutes to choose your badge when creating an account, now that should be extended to 1 hour)